specs/am/architecture-management-vocab.html

```diff
@@ -38,7 +38,7 @@
           },
         ],
 
-        //publishDate:  "2018-10-09",
+        publishDate: "2020-08-31",
```

Comment:
That would be a challenge given that the vote has not even opened yet and I did not see the ZIP of the artifacts.

Reply:
commented out, will set when vote completes and before creating Zip file.

Reply:
The right sequence should be: create a PR and uncomment, create ZIP, initiate a vote, merge when passed.